### ISSUE TITLE
DJ Hero 2 doesn't recognize the DJ Hero dongle while the original does

### ISSUE THREAD

**Issue by marysiamzawka** [RPCS3/rpcs3#12767]
### Quick summary

When I have the DJ Hero dongle plugged in, DJ Hero 2 doesn't recognize any inputs, while DJ Hero 1 talks to the dongle just fine.

### Details

Tested on Arch Linux with the following udev rule set for the dongle:
```
/etc/udev/rules.d/99-ds3-controllers.rules

# DJ Hero dongle
SUBSYSTEM=="usb", ATTRS{idVendor}=="12ba", ATTRS{idProduct}=="0140", MODE="0666"
```

Games I have been testing with:
DJ Hero (BLES00602)
DJ Hero 2 (BLES00896)

Both games detect the dongle and the controller fine on a real PS3.
On my instance of RPCS3 only DJH1 detects the dongle while DJH2 doesn't react to it.

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9733217/RPCS3.log.gz)

**Comment by RipleyTom** [comment]
You have emulated turntable turned on in I/O:
  Turntable emulated controller: 1 controller

You need to edit the custom config and set DJ Hero Emulated Turntable to Null in I/O Tab.

**Comment by marysiamzawka** [comment]
Changing that did not make a difference - DJH1 still picks up the turntable while DJH2 doesn't.
Including a [video](https://www.youtube.com/shorts/j2O_v2jYrKg) and a log below
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9757270/RPCS3.log.gz)

**Comment by RipleyTom** [comment]
Reopening, I'll take a look later.
If you want to help you can use previous builds page on rpcs3.net to find the build that broke it(because it used to work for sure).

**Comment by RipleyTom** [comment]
Tried and I wasn't able to reproduce, works fine like it always has for me.

**Comment by archonpy** [comment]
Will try to get logs and more information when I get back to my machine. The problem exists at least for a subset of people based on what I’ve read on the discord channel. For me I tried on several Windows 10 pcs (with varying updates, including one with the latest win 10 updates). Like the original ticket, I also tried several past builds dating back to when it was first supported. The ps3 turntable works perfect with Dj Hero 1. For Dj Hero 2 it has never worked for me, regardless of emulated turntable set to null or not and even with just single player. Appreciate all the work the rpcs3 contributors have made, it’s an awesome project and I will try to get the logs and post it.
